lib/hope-tab.js

```diff
@@ -58,6 +59,47 @@ window.HopeTab = fxosComponent.register('hope-tab', {
     }
   },
 
+  onError(e) {
+    debug('on error', e, this.els.input.value, this.userSubmittedUrl);
+    var networkErrors = [
+      'dnsNotFound',
+      'connectionFailure'
```

Comment:
Can we reset `userSubmittedUrl` to null in the `onLocationChange` handler to guarantee this?